packages/wrapped-keys-lit-actions/tsconfig.json

```
@@ -12,7 +12,7 @@
     "checkJs": false
   },
   "files": [],
-  "include": [],
+  "include": ["global.d.ts", "node_modules/ethers/lib/ethers.d.ts"],
```

```suggestion
  "include": ["global.d.ts"],
```

This actually might not be needed as we are "importing" the `ethers` directly to the Lit Action code.